### PR TITLE
Add 1.8.9 to release notes

### PR DESCRIPTION
**Pull request by mluu12**
Closes https://github.com/influxdata/docs-v2/issues/2965

Added 1.8.8 section.

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md [hunk 1]
```diff
@@ -9,6 +9,17 @@ menu:
...
+The InfluxDB Enterprise 1.8.8 release builds on the InfluxDB OSS 1.8.8 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v188-2021-08-0?).
+
+### Bug fixes 
+
+- Prevent silently dropped writes with overlapping shards.
+- Restore portable backup bug.
+- Systemd-startup script should be executable by group and others.
+
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:22]: @mluu12, if lines 19-21 are the InfluxDB OSS 1.8.8 fixes, then we can delete them here

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md [hunk 2]
```diff
@@ -9,6 +9,17 @@ menu:
     parent: About the project
 ---
 
+## v1.8.8 [2021-08-03]
+The InfluxDB Enterprise 1.8.8 release builds on the InfluxDB OSS 1.8.8 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v188-2021-08-0?).
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:15]: and to be sure, we'll want to update this link to the actual release data

**Comment by timhallinflux** [comment]
I think there was a problem with the 1.8.8 build and we are on 1.8.9 now.

**Comment by mluu12** [comment]
@timhallinflux Yes! The content should be updated to 1.8.9, but I forgot to change the title from 1.8.8 to 9. My mistake!

**Review by kelseiv — APPROVED** [review]

**Pull request merged**